### PR TITLE
Callouts for upcoming Microsoft adapter support

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Add callouts about dbt Cloud support for MS Fabric and Synapse coming soon to these 3 pages:
- https://docs.getdbt.com/docs/supported-data-platforms
- https://docs.getdbt.com/docs/verified-adapters
- https://docs.getdbt.com/docs/cloud/connect-data-platform/about-connections 

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Need review from PMM

Adding new pages (delete if not applicable):
- [x] ~~Add page to `website/sidebars.js`~~ new file is a partial
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/snippets/_microsoft-adapters-soon.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+:::tip Coming soon
+dbt Cloud support for the Microsoft Fabric and Microsoft Synapse adapters is coming soon!
```
Comment (dataders) [inline website/snippets/_microsoft-adapters-soon.md]: "coming soon" felt redundant given it's on the header for callout. also played w/ the language to make it more direct and clear? take it or leave it
```suggestion
Support for the Microsoft Fabric and Azure Synapse in dbt Cloud!
```

image

**Review by dataders — APPROVED** [review]
@nghi-ly thank you I'm fine with whatever language you land on. this is great way to bridge the gap imho.

one extra credit thing might be to add this call out to both:
- [fabric-setup.md](https://docs.getdbt.com/docs/core/connect-data-platform/fabric-setup) and
- [azuresynapse-setup.md](https://docs.getdbt.com/docs/core/connect-data-platform/azuresynapse-setup)

**Comment by nghi-ly** [comment]
thank you @dataders ! good callout about the adapter pages. lemme check with the PMMs 👍

File: website/snippets/_microsoft-adapters-soon.md (hunk 1 above)
Comment (nghi-ly) [inline website/snippets/_microsoft-adapters-soon.md]:
```suggestion
dbt Cloud support for the Microsoft Fabric and Azure Synapse Analytics adapters is coming soon! 
```

**Pull request merged**